core/domain/classifier_services.py

```diff
@@ -124,6 +124,57 @@ def classify_string_classifier_rule(state, normalized_answer):
 
     return None
 
+def train(exploration):
```

Comment:
@seanlip you are right, earlier I was thinking that we could place a temporary classifier (like naive bayes) instead of LDAStringClassifier, but, then we decided that it would be a waste of efforts which could be better utilized in some other way. @pranavsid98 thanks for this PR, I don't think we will not be using it, instead we would have to wait till the necessary things have been implemented.